doc/dev/decisionrecords/APIGraphQLDesign.md

```diff
@@ -28,6 +28,18 @@ new features, consider what is best; To follow the existing style or follow the
 
 ### Context and Problem Statement
 
-Our APIs need to be backwards compatible as they are used by hundreds of clients (web-pages/apps/services)
-Correcting mistakes may not be possible or may take a long time. 
+Our APIs need to be backwards compatible as they are used by hundreds of clients
+(web-pages/apps/services). Correcting mistakes may not be possible or may take a long time.
 
+We allow breaking API changes in these cases:
+
+- During incremental development of a new feature (multiple PRs). The new part of the API should be
+  documented as _"WorkInProgress - Why/When is it ready"_. A new API should not stay "open" for a
+  long time.
+- Deprecated features can be removed after 2 years (4 minor releases) unless a user wants to keep 
+  it. We will revert a removal if the request to keep it comes after the feature is removed. 
+  Example:
+
+> Use featurer xyz instead. 
+> Note! This feature is subject for removal from september 2027! If you want this to stay, pleace
+>       notify the OTP community.
```

Comment:
```suggestion
> Note! This feature is subject for removal from in September 2027! If you want this to stay, please
>       notify the OTP community.
```

Reply:
Should we make it a little clearer that not _anyone_ can show up and prevent removal of something? You'd have to build a relationship and contribute in some way if you want to steer the OTP developers.

Reply:
@leonardehrenfried I slightly modified your suggestion as I think u accidentally left in the "from" before "in"

Reply:
I ment _ anyone_, the reason is that we would like to give people a guarantee, of cause if they do not tell us _before_ we remove a feature, we can request them to do the work of adding it back in - I think that is fare enough. 

We should probably mention this at the next PLC meeting.